### PR TITLE
new mod(win-d-per-monitor)

### PR DESCRIPTION
**Pull request by easyatm**

### CODE REVIEW

File: mods/win-d-per-monitor.wh.cpp [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+    int size_needed = MultiByteToWideChar(CP_ACP, 0, &str[0], (int)str.size(), NULL, 0);
+    std::wstring wstrTo(size_needed, 0);
+    MultiByteToWideChar(CP_ACP, 0, &str[0], (int)str.size(), &wstrTo[0], size_needed);
+    return wstrTo;
+}
+#define log(...)                                                                                                                                               \
+    {                                                                                                                                                          \
+        std::string gb = std::format(__VA_ARGS__);                                                                                                             \
+        auto ws = ansi_unicode(gb);                                                                                                                            \
+        Wh_Log(L"%s", ws.c_str());                                                                                                                             \
```
Comment (m417z) [inline mods/win-d-per-monitor.wh.cpp]: FYI, you can just use `Wh_Log(L"%S", gb.c_str());` here. Or even better:
```cpp
Wh_Log(L"%S", std::format(__VA_ARGS__).c_str());
```
Then the string will only be constructed if debugging is enabled.

File: mods/win-d-per-monitor.wh.cpp [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+            listWnd.clear();
+            activeWnd(hLastWnd);
+        }
+
+        return true;
+    }
+};
+
+int (*__cdecl HandleGlobalHotkey_Original)(unsigned __int64, __int64);
```
Comment (m417z) [inline mods/win-d-per-monitor.wh.cpp]: Did you mean:
```suggestion
void (*__cdecl HandleGlobalHotkey_Original)(unsigned __int64, __int64);
```

**Review by m417z — COMMENTED** [review]
Looks good, I left a couple of minor comments. Also please fix the GitHub check failures (missing author name and hooks variable name).

**Comment by easyatm** [comment]
The remote module's description is incorrectly localized, while the local module displays Chinese correctly.

image

image

image

**Comment by m417z** [comment]
The remote data is always non-localized, I have it on my list to implement it in one of the future updates.

**Pull request merged**